charts/README.md

```diff
@@ -0,0 +1,94 @@
+# Install with Helm 
+
+## Deploy cert-manager chart
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --version v1.12.0 \
+  --set installCRDs=true
+```
```

Comment:
Would it be more suitable to define this chart as sub-chart dependency.

Helm allows you to do this via: https://v2.helm.sh/docs/developing_charts/#chart-dependencies

You can also make this conditional, so users that are bringing/using their own `cert-manager` won't install it twice.

```yaml
dependencies:
  - name: cert-manager
    version: 1.12.0
    repository: https://charts.jetstack.io
    condition: cert-manager.install
```

Looks like the cert-manager chart also allows specification of the namespace for use as a sub-chart:
https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L155

values.yaml
```yaml
...
cert-manager:
  install: true
  namespace: cert-manager
...
```

Happy to include this as a followup to these changes, to not overcomplicate things here :)